dev/ci/ci-compcert.sh

```diff
@@ -5,6 +5,10 @@ ci_dir="$(dirname "$0")"
 
 CompCert_CI_DIR="${CI_BUILD_DIR}/CompCert"
 
+# Temporal workaround, to be removed when upstream decides what to do
```

Comment:
Doesn't "temporal" mean some church-state concept? And something about brain parts.

Reply:
I will never learn.